niri-config/src/lib.rs

```diff
@@ -977,6 +977,8 @@ pub enum Action {
     MoveColumnRight,
     MoveColumnToFirst,
     MoveColumnToLast,
+    MoveColumnLeftOrMonitorLeft,
+    MoveColumnRightOrMonitorRight,
```

Comment:
```suggestion
    MoveColumnLeftOrToMonitorLeft,
    MoveColumnRightOrToMonitorRight,
```
For grammar and consistency sake (like i.e. `MoveWindowDownOrToWorkspaceDown`)